libtelemetry/src/main/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -241,6 +241,10 @@ boolean optLocationOut() {
     return isLocationOpted;
   }
 
+  EventsQueue getQueue() {
```

Comment:
what is the benefit of exposing/relaxing access restriction to queue object? it seems you just want to check if queue is empty, can we instead expose `boolean isQueueEmpty()` api?